# Request: No need for a fill
---
# Response: A vector of floats to fill and return 
float32[3] orientation
